demos/hbase-hdfs-load-cycling-data/distcp-cycling-data.yaml

```
@@ -11,14 +11,15 @@ spec:
           # We use 24.3.0 here which contains the distcp MapReduce components
           # This is not included in the 24.7 images and will fail.
           # See: https://github.com/stackabletech/docker-images/issues/793
-          image: docker.stackable.tech/stackable/hadoop:3.3.4-stackable24.3.0
+          image: docker.stackable.tech/stackable/hadoop:3.4.0-stackable24.11.0
```

I think this should be:

```suggestion
          image: docker.stackable.tech/stackable/hadoop:3.4.0-stackable24.7.0
```

Because 24.11 is not available yet, and this branch gets merged before that.

---

Ah ok, in that case we need to go 0.0.0-dev (@sbernauer is gonna love this, but it won't be for long :smile:).

```suggestion
          image: docker.stackable.tech/stackable/hadoop:3.4.0-stackable0.0.0-dev
```